andela/workdey
# #[122978061] Search for available needs

#### What does this PR do?

This PR ensures that a taskee can search for available tasks that has not yet being assigned and that has been Posted
#### How should this be manually tested?
- Log in as a Taskee
- click the search Icon on the Nav bar
- Enter the skill you want
- then you can see a list of Jobs that are available based on the skill you've entered
#### What are the relevant pivotal tracker stories?
#122978061
#### Screenshots (if appropriate)

![screen shot 2016-07-21 at 13 53 16](https://cloud.githubusercontent.com/assets/14159384/17023437/48e661b0-4f4b-11e6-8eee-8f685271eeac.png)
![screen shot 2016-07-21 at 13 52 48](https://cloud.githubusercontent.com/assets/14159384/17023439/4e10821a-4f4b-11e6-988e-03ad9795c80f.png)
![screen shot 2016-07-21 at 14 00 14](https://cloud.githubusercontent.com/assets/14159384/17023490/847b7f8a-4f4b-11e6-9c65-1b2b9dc1c991.png)
